app-backend/datamodel/src/main/scala/Campaign.scala

```diff
@@ -18,7 +18,9 @@ final case class Campaign(
     partnerLogo: Option[String] = None,
     parentCampaignId: Option[UUID] = None,
     continent: Option[Continent] = None,
-    tags: List[String] = List.empty
+    tags: List[String] = List.empty,
+    childrenCount: Int,
+    status: Map[String, Int]
```

Comment:
I think it's true that everywhere else we have a `status` field it's a string enum. It would be clearer to give this some kind of name indicating its plurality, like `project_statuses` or `project_status_counts` (similar to `task_status_summary` on `annotation_projects`) with corresponding DB changes.